skyportal/handlers/api/group.py

```diff
@@ -285,12 +285,12 @@ def post(self):
 
         g = Group(name=data["name"], nickname=data.get("nickname") or None)
         DBSession().add(g)
-        self.verify_permissions()
+        self.verify_and_commit()
```

Comment:
This should still be `verify_permissions`

Reply:
Ah yes, will push up a version with L288-289 removed